### PR TITLE
Added crt mode to the funbox

### PR DESCRIPTION
**Pull request by fehmer**
This PR adds a CRT mode to the funbox which emulates an old CRT monitor by adding scanlines, a background glow. The mode does not effect the colors from the theme.

Theme Serika dark:
image

Theme Serika:
image

### CODE REVIEW

**Comment by Miodec** [comment]
Aaaand it also overlaps any custom background images you have.

**Comment by fehmer** [comment]
> Styling on the center content element causes overlap of the timer bar. And I don't really like the left and right borders that it adds.
> Aaaand it also overlaps any custom background images you have.

good point, Moved the screen glow to the body which gets replaced when using custom background image. Also removed the borders.

**Comment by Miodec** [comment]
There is a white flash coming from somewhere when restarting the test

**Comment by Miodec** [comment]
The scanlines dissapear when leaving the typing page.

**Comment by fehmer** [comment]
fixed

**Review by Perseus333 — APPROVED** [review]
I really like this theme. It should be in the main website.

File: frontend/src/styles/core.scss [hunk 1]
```diff
@@ -532,3 +533,58 @@ key {
...
+  height: 100px;
+  z-index: 1001;
+  position: absolute;
+  bottom: 100%;
+  opacity: 0.1;
+  pointer-events: none;
+}
+
+body.crtmode {
+  text-shadow: 3px 0 1px rgba(0, 30, 255, 0.5), -3px 0 1px rgba(255, 0, 80, 0.3),
```
Comment (Miodec) [inline frontend/src/styles/core.scss:548]: I think this shadow maybe should be based on the theme colors and not hard coded? Right now I think it only fits some themes but not others.

**Pull request merged**